### PR TITLE
Load perf

### PR DESCRIPTION
**Pull request by AlbertoElias**

### CODE REVIEW

File: src/js/ads.js [hunk 1]
```diff
@@ -96,8 +95,13 @@ class VideoAds {
...
 		this.playAdEventHandler = this.playAdEventHandler.bind(this);
-		this.overlayEl && this.overlayEl.addEventListener('click', this.playAdEventHandler);
-		this.video.placeholderEl && this.video.placeholderEl.addEventListener('click', this.playAdEventHandler);
+		if (this.video.opts.placeholder) {
+			this.playAdEventHandler();
+		} else {
+			this.overlayEl && this.overlayEl.addEventListener('click', this.playAdEventHandler);
+		}
+
+		// this.video.placeholderEl && this.video.placeholderEl.addEventListener('click', this.playAdEventHandler);
```
Comment (ironsidevsquincy) [inline src/js/ads.js]: can be removed

File: src/js/ads.js [hunk 2]
```diff
@@ -182,7 +186,7 @@ class VideoAds {
 		this.overlayEl && this.overlayEl.removeEventListener('click', this.playAdEventHandler);
 		this.overlayEl && this.video.containerEl.removeChild(this.overlayEl);
 		delete this.overlayEl;
-		this.video.placeholderEl && this.video.placeholderEl.removeEventListener('click', this.playAdEventHandler);
+		// this.video.placeholderEl && this.video.placeholderEl.removeEventListener('click', this.playAdEventHandler);
```
Comment (ironsidevsquincy) [inline src/js/ads.js]: ditto

**Comment by ironsidevsquincy** [comment]
Once tests are passing, 👍

**Comment by AlbertoElias** [comment]
Thanks! All fixed, I'll merge and release when Travis finishes

**Pull request merged**